tests/fixtures/integration/actions/ee_details/test_stdout_direct_ee.py/test/0.json

```diff
@@ -0,0 +1,84 @@
+    "index": 0,
+    "comment": "Playbook output",
+    "output": [
+        "PLAY [Gather execution environment details for quay.io/ansible/ansible-runner:devel] *******************************************************************************************************************",
+        "TASK [Gather execution environment details] ************************************************************************************************************************************************************",
+        "TASK [ansible.ansible_navigator.ee_details : Gather collection details] ********************************************************************************************************************************",
+        "ok: [execution-environment]",
+        "TASK [ansible.ansible_navigator.ee_details : Gather operating system details] **************************************************************************************************************************",
+        "ok: [execution-environment]",
+        "TASK [ansible.ansible_navigator.ee_details : Gather python package details] ****************************************************************************************************************************",
+        "ok: [execution-environment]",
+        "TASK [ansible.ansible_navigator.ee_details : Extract collections] **************************************************************************************************************************************",
+        "ok: [execution-environment]",
+        "TASK [ansible.ansible_navigator.ee_details : Extract python packages] **********************************************************************************************************************************",
+        "ok: [execution-environment]",
+        "PLAY [Extract execution environment details for quay.io/ansible/ansible-runner:devel] ******************************************************************************************************************",
+        "TASK [Extract execution environment details] ***********************************************************************************************************************************************************",
+        "TASK [ansible.ansible_navigator.ee_report : Summarize the execution environment details] ***************************************************************************************************************",
+        "ok: [execution-environment]",
+        "PLAY [Review execution environment details for quay.io/ansible/ansible-runner:devel] *******************************************************************************************************************",
+        "TASK [Review execution environment details] ************************************************************************************************************************************************************",
+        "ok: [execution-environment] => {",
+        "    \"summary\": {",
+        "        \"ansible_version\": \"2.12.0.dev0\",",
```

Comment:
Since the fixture is for `ansible-runner:devel` the value in the summary field are subject to change. Maybe can use a specific version of ansible-runner EE

Reply:
in the test we are just looking for a value on the screen..... at the bottom.  I would like to have the "what" we were looking for in the fixture tho